### ISSUE TITLE
Itinerary between two nodes?

### ISSUE THREAD

**Issue by nclm** [CyberCRI/KnowNodes#115]
To be able to find a path between two arbitrary nodes, if enough links are available.

Let's say I want to find a way between [Douglas Adams] and [Wes Anderson]. Like on a maps website, I could get an itinerary: [Douglas Adams] wrote [The Hitchhicker’s Guide to the Galaxy (novel)] was adapted into [The Hitchhicker’s Guide to the Galaxy (film)] had a character played by [Mos Def] played in [Be Kind Rewind] was directed by [Michel Gondry] directed [Eternal Sunshine of the Spotless Mind] had a characted played by [Kirsten Dunst] played in [Virgin Suicide] was directed by [Sofia Coppola] is the cousin of [Jason Schwartzman] played in [Rushmore] was directed by [Wes Anderson]. You have reach your destination. It was not the shortest way, but anyway, I feature like this would be interesting!

**Comment by Garbash** [comment]
Good idea! It's also pretty easy to implement with our data structure!

On Thu, Nov 7, 2013 at 8:34 AM, nicolas notifications@github.com wrote:

> To be able to find a path between two arbitrary nodes, if enough links are
> available. Let's say I want to find a way between [Douglas Adams] and [Wes
> Anderson]. Like a maps website, I could got an itinerary: [Douglas Adams]
> wrote [The Hitchhicker’s Guide to the Galaxy (novel)] was adapted into [The
> Hitchhicker’s Guide to the Galaxy (film)] had a character played by [Mos
> Def] played in [Be Kind Rewind] was directed by [Michel Gondry] directed
> [Eternal Sunshine of the Spotless Mind] had a characted played by [Kirsten
> Dunst] played in [Virgin Suicide] was directed by [Sofia Coppola] is the
> cousin of [Jason Schwartzman] played in [Rushmore] was directed by [Wes
> Anderson]. You have reach your destination. It was not the shortest way,
> but anyway, I feature like this would be interresting!
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/CyberCRI/KnowNodes/issues/115
> .

## 

Get some insight of what makes me tick:
My twitter https://twitter.com/#!/garbash
Knownodes mailing list https://groups.google.com/forum/#!forum/knownodes

Dor Garbash
108 Rue du faubourg du temple, Paris 75011
+33-652919878
